### DISCUSSION TITLE
Z2M configuration not persistant after restart

### DISCUSSION THREAD

**Discussion by sharpmindstudio** [Koenkk/zigbee2mqtt#11290] [General]
Hi there,
I need some help before I go nuts here:  I try to make a settings in z2m persistant but after I restart the VM the settings goes back to default.(true)
The settings I'm talking about is:  homeassistant: legacy_triggers: false
I have tried to make the change in add-on configuration directly, in the config\zigbee2mqtt\configuration.yaml, and in the gui itself.
I'm using VMware, ha 2022.2.3, Zigbee2mqtt 1.23.0-1
Nothing interesting in the logs

**Comment by JaegermeisterS** [comment]
Hi
I am experiencing exactly the same issue.  HA 2022.2.2, z2mqtt 1.23.0-1.
BR Stefan

**Comment by sharpmindstudio** [comment]
I think this is reseting the setting I'm tring to set : "INFO: Adjusting Zigbee2mqtt core yaml config with add-on quirks ...".
But I have no clue where is the default configuration is coming from :(

**Comment by sharpmindstudio** [comment]
What I want to be persistant is something like this (config/zigbee2mqtt/configuration.yaml):
"
homeassistant:
  legacy_entity_attributes: false
  legacy_triggers: false
permit_join: false
"
and after restart I get the default:   homeassistant: true

**Comment by m0wlheld** [comment]
Must read:

```
homeassistant:
  legacy_entity_attributes: false
  legacy_triggers: false
permit_join: false
```

**Reply by sharpmindstudio** [reply]
Yes this is how I try to save it (the indentation was autoremoved). The issue is that the save goes back to default after VM restart.
